packages/wrangler/src/type-generation/runtime/index.ts

```diff
@@ -0,0 +1,100 @@
+import { readFileSync } from "fs";
+import { writeFile } from "fs/promises";
+import { Miniflare } from "miniflare";
+import { ensureDirectoryExists } from "../../utils/log-file";
```

Comment:
This file doesn't really have anything to do with writing a log-file, I'm not sure it should necessarily be importing from here. It's a pretty simple utility, so maybe it could just be inlined? Or extracted out to it's own file independent of `log-file`?

Reply:
I'll move it to a new `packageswrangler/src/utils/filesystem.ts` util file.